app/code/Magento/InventorySales/composer.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Also, 
1) need to add the dependency in the global composer. json (after that need to regenerate composer lock)
2) Need to introduce one more module like InventorySalesApi